src/seedu/addressbook/data/person/Address.java

```diff
@@ -36,24 +35,4 @@ public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
 
-    @Override
-    public String toString() {
-        return value;
-    }
-
-    @Override
-    public boolean equals(Object other) {
```

Comment:
Note that now an Email and Address having the same value will be considered equal.